knpu/intro.md

```diff
@@ -56,4 +56,4 @@ php app.php
 You'll see two emails are sent to two lucky people.
 
 
-[1]: knpuniversity.com/composer
+[1]: knpuniversity.com/screencast/composer
```

Comment:
Hey @Narendra23 ,

Great catch! Please, could you also force HTTPS here? We need to tweak this link to `https://knpuniversity.com/screencast/composer` one and I'll merge it immediatelly ;)